tests/Unit/IO/H5/Test_H5File.cpp

```diff
@@ -39,7 +39,17 @@ void test_core_functionality() {
     file_system::rm(h5_file_name, true);
   }
   // [h5file_readwrite_get_header]
-  const std::string input_source_yaml{"TestOption: 4\n\nOtherOption: 5\n\n"};
+  std::string input_source_yaml{"TestOption: 4\n\nOtherOption: 5\n\n"};
+  // Add a giant set of comments to the h5 file, to test that we can
+  // read and write very long yaml files to H5 attributes
+  input_source_yaml += "# ";
+  for (size_t i = 0; i < 1000000; ++i) {
+    input_source_yaml += "abcdefghij"s;
+    if (i % 100) {
```

Comment:
No, I meant it as written...I insert line breaks every 1000 characters deliberately. If you insist that I change this, sure, but this is as I intended. I wanted to have a comment spanning many lines, to simulate a user adding many lines to the file, instead of just one long line.

I squashed. @wthrowe do you still want me to make this change?

Reply:
@wthrowe I clarified my intention with a comment, and I also modified this line to keep line widths below 80 characters, as we would in a real H5 file. Does this look ok to you now?

Reply:
I have no problems with that, including with long lines, but your code isn't doing what you think it is.  The generated comment looks like
```
# abcdefghijabcdefghij
# abcdefghij
# abcdefghij
# abcdefghij
# abcdefghij
# abcdefghij
# abcdefghijabcdefghij
# abcdefghij
# abcdefghij
# abcdefghij
# abcdefghij
# abcdefghij
# abcdefghijabcdefghij
# abcdefghij
```

Reply:
I get it now...my mistake! I fixed this as you suggested @wthrowe and squashed...thanks for catching this silly mistake